packages/soxr/meta.yaml

```diff
@@ -0,0 +1,25 @@
+package:
+  name: soxr
+  version: 0.5.0.post1
+  top-level:
+    - soxr
+source:
+  url: https://files.pythonhosted.org/packages/02/c0/4429bf9b3be10e749149e286aa5c53775399ec62891c6b970456c6dca325/soxr-0.5.0.post1.tar.gz
+  sha256: 7092b9f3e8a416044e1fa138c8172520757179763b85dc53aa9504f4813cff73
+  patches:
+    - patches/0001-Fix-function-signatures.patch
+requirements:
+  run:
+    - numpy
+build:
+  script: |
+    # Include the tests in the wheel for our test_soxr.py (will be vendored into soxr-tests)
+    sed -i '/\[tool\.scikit-build\]/a wheel.packages = {"soxr" = "src\/soxr", "soxr\/tests" = "tests"}' pyproject.toml
```

Comment:
For the same reason as the previous comment, Let's not include tests if possible.